lib/utils/filename-normaliser.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const path = require('path');
+const findup = require('find-up');
+const { existsSync } = require('fs');
+
+const pkg = require(findup.sync('package.json'));
+
+const normaliseFileName = function(fileName = '') {
+  // check for invalid valid file name
+  // for example,
+  // path.basename('') will be equal to fileName
+  // path.basename('base/file') will not be equal to fileName
```

Comment:
I still don't understand this. can't we check for `fileName === ''` directly then?

Reply:
I have updated the comments, please take a look now.